app/controllers/photos.js

```diff
@@ -0,0 +1,7 @@
+const { getPhotos, getPhotosByAlbum } = require('../services/albums');
+
+exports.getPhotos = (req, res, next) => {
+  const { albumId } = req.query;
+  const callbackAction = albumId ? getPhotosByAlbum(albumId) : getPhotos();
+  callbackAction.then(photos => res.send({ photos })).catch(next);
```

Comment:
again, no need to encase the response in an empty objects, a simple list is fine